bundle/manifests/pulp-operator.clusterserviceversion.yaml

```diff
@@ -1089,7 +1089,7 @@ spec:
                 - name: RELATED_IMAGE_PULP_REDIS
                   value: redis:latest
                 - name: RELATED_IMAGE_PULP_POSTGRES
-                  value: postgres:13
+                  value: quay.io/sclorg/postgresql-15-c9s:latest
```

Comment:
Hi @rooftopcellist!

Thank you for your contribution!
By default, this new image will run as user `1000` in vanilla k8s. Because of that, when the bkp-manager container tries to make a copy of `/var/lib/pulp/tmp/<worker files>` (which has 0700 permission and is owned by user `700`), it is getting `permission denied` error and failing the backup task.

The old image (postgres:13 from dockerhub) had the access permissions because it was running with uid 0 (root).